pkg/cluster/internal/providers/docker/network_test.go

```diff
@@ -18,9 +18,65 @@ package docker
 
 import (
 	"fmt"
+	"regexp"
+	"strings"
 	"testing"
+
+	"sigs.k8s.io/kind/pkg/errors"
+	"sigs.k8s.io/kind/pkg/exec"
+
+	"sigs.k8s.io/kind/pkg/internal/integration"
 )
 
+func TestIntegrationEnsureNetworkConcurrent(t *testing.T) {
```

Comment:
We can consider reworking the make targets based on this pattern.

to run only "unit": `go test -short ...`
to run only "integration": `go test -run ^TestIntegration ...`